src/api/newsletters.ts

```diff
@@ -19,11 +11,17 @@ const getSheetsNewsletters = () => {
 			frequency: 'Weekly, Friday/ad usually, ~12-2pm',
```

Comment:
Out of scope, but this frequency doesn't look like something that anyone could use (except as display text). Does this suggest we need `frequency` _and_ `frequencyDisplayText` (or similar name)?